src/main/kotlin/domain/BlackjackCards.kt

```diff
@@ -0,0 +1,29 @@
+package domain
+
+class BlackjackCards() {
+    val cards: MutableList<Card> = mutableListOf()
+
+    companion object {
+        const val BLACKJACK_MAX_SCORE = 21
+    }
+
+    fun receiveCard(card: Card) {
+        cards.add(card)
+    }
+
+    fun calculateScore(): Int {
+        var total = cards.sumOf { it.value.basicScore }
+        var aceCount = cards.count { it.value == CardValue.ACE }
+
+        while (total > BLACKJACK_MAX_SCORE && aceCount > 0) {
+            total -= 10 // ACE 11 → 1 변경
+            aceCount--
+        }
+
+        return total
+    }
+
+    fun displayCardInfo(): String {
+        return cards.joinToString(", ") { it.toString() }
```

Comment:
요렇게만 써도 결과는 동일한 것 같아요~~
```suggestion
        return cards.joinToString()
```

Reply:
근데 써주신 구조가 뭔가 코틀린스럽고 특이해서 살펴보면서 joinToString 함수에 대해 더 학습할 수 있었어요!
감사합니다 👍

Reply:
저도 내부를 보니 어차피 다 생략할 수 있는 것이었군요..! 감사합니다